README.md

```diff
@@ -700,6 +701,25 @@ interface Table extends Parent {
 
 **Table** represents 2d data.
 
+### CustomCodeComponent
+
+```ts
+interface CustomCodeComponent extends Parent {
+  type: "custom-code-component"
+  path: string
+  versionRange: string
+  lastModified: string
+  dataIdentifier: string
+  attributes: {
+    [key: string]: string | boolean | undefined
+  }
+  children: (ImageSet | Paragraph | CustomCodeComponent)[]
+}
```

Comment:
Suggestion: We should document the meaning of each field. This would be beneficial in the long term also for other nodes. I'll try some suggestions to kick off the discussion, but @joannaskao @emmalewis @aendra-rininsland could help, too.
Suggestions are based on the [TDD](https://coda.io/d/Custom-Components_dR6le1je5Z0/TDD-Phase-1_suUmD#_lu1wp) 

```suggestion
interface CustomCodeComponent extends Parent {
  type: "custom-code-component"
  // Repository for the code of the component in the format "[github org]/[github repo]/[component name]".
  path: string
  // Semantic version of the code of the component, e.g. "^0.3.5".
  versionRange: string
  // Last date-time where the instance of this block was modified, in ISO-8601 format.
  lastModified: string
  // A unique identifier to track component metrics in analytics.
  dataIdentifier: string
  // Configuration data to be passed to the component.
  attributes: {
    [key: string]: string | boolean | undefined
  }
  children: (ImageSet | Paragraph | CustomCodeComponent)[]
}
```

Reply:
yep, good idea. there's no duplication necessary though. the .d.ts and the json schemas are all built automatically from the readme just like donald knuth wanted

so if you change these `// ` comments to `/** */` comments (jsdoc) they'll be kept in the `.d.ts` file and they'll also be used at the `"description"` field in the generated json schema

Reply:
Thank you @chee . I updated my suggestion.

Reply:
Thanks both 🙏  Happy for me to commit this suggestion @umbobabo ?